modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -35,8 +37,10 @@ abstract class ParagraphsTestBase extends AbstractKernelTestBase {
     'oe_media',
     'oe_media_avportal',
     'oe_media_iframe',
+    'oe_multilingual',
```

Comment:
Not a fan of this, and not a fan of the base paragraph test at all as we are adding all the modules all the time and we cannot catch the paragraphs in isolation. At this point the kernel test is useless, and we should just use a browser test... Let's avoid to add these modules here. Let's keep it like the document paragraph test, without requiring oe_multilingual.